generators/generator-base.js

```diff
@@ -2238,6 +2238,9 @@ module.exports = class JHipsterBaseGenerator extends PrivateBase {
         if (options.recreateInitialChangelog) {
             this.configOptions.recreateInitialChangelog = options.recreateInitialChangelog;
         }
+        if (options.withAdminUi) {
+            this.configOptions.withAdminUi = options.withAdminUi;
```

Comment:
```suggestion
        if (options.withAdminUi !== undefined) {
            this.jhipsterConfig.withAdminUi = options.withAdminUi;
```
For reproducibility.

Reply:
Done, what is the difference between configOptions and jhipsterConfig?